webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadMenu.vue

```diff
@@ -52,14 +52,18 @@ export default {
   data: () => ({
     menu: false,
     loading: false,
+    transferRulesDownloadStatus: false,
   }),
   computed: {
     menuButtonStyle() {
       return this.$vuetify && this.$vuetify.rtl && 'top:8px;left:4px;' || 'top:8px;right:4px;';
     },
-    enabledActions() {
+    actions() {
```

Comment:
Can you explain what is "actions" stand for ? I'm not sure to understand.
Thanks

Reply:
can you show me where "actions()" functione is used ?